pysb/core.py

```diff
@@ -1267,20 +1268,31 @@ class Parameter(Component, Symbol):
         The numerical value of the parameter. Defaults to 0.0 if not specified.
         The provided value is converted to a float before being stored, so any
         value that cannot be coerced to a float will trigger an exception.
+    nonnegative : bool, optional
```

Comment:
Note that the Attributes section below currently says "Identical to Parameters", but these new `nonnegative` and `integer` parameters are not stored as attributes. Maybe that text just needs to changed to something like "Identical to Parameters, excluding \`nonnegative\` and \`integer\`", or it could say "value (see Parameters above)".

Reply:
done